src/components/App/App.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Feed from '../Feed/Feed';
 import Form from '../Form/Form';
+import {BrowserRouter as Router, Route, Switch} from 'react-router-dom';
 import './App.css';
 
 interface Multimedia {
```

Comment:
For the Article interface, you might have to use this elsewhere in your application (which is probably why you have exported it here). I've seen other React apps where they use a "Models.tsx" file in the src directory that holds most of their interfaces (expect for interfaces/types used for props in components).

That way you can import the Models file and have your interfaces live in that file. Otherwise, you if you need to reference the Article type in another file, you'd be importing Article from App, and that might seem weird for something to be imported into a component from the App component.